packages/storage/src/indexed-db/index.ts

```diff
@@ -812,11 +816,7 @@ export class IndexedDb implements IndexedDbInterface {
   }
 
   fetchStakingTokenId(): AssetId {
-    const registryClient = new ChainRegistryClient();
-    const registry = registryClient.get(this.chainId);
-    const stakingToken = registry.stakingAssetId;
-
-    return stakingToken;
+    return this.stakingTokenAssetId;
```

Comment:
Probably `fetch` is the wrong name for this. Why don't we just make this a public variable so folks can grab it from the instance: https://github.com/penumbra-zone/web/pull/1367/files#diff-8298d111bc94531ec943d5f024a9ab57657ad28670caea51e3aa9eda88bf3954R83

Reply:
Also, if this is done, we should make storage a major version bump given we change the API in a breaking way

Reply:
@grod220 will you be able to publish packages to npm?